src/infra/dataverse/sparql/sparqlGateway.ts

```diff
@@ -56,6 +59,7 @@ export const sparqlGateway: DataversePort = {
         }
         ${byType === 'all' ? '' : `FILTER ( ${byTypeFilter(byType)} )`}
         ${byProperty?.property ? `FILTER ( ${byPropertyFilter(byProperty)} )` : ''}
+        ${O.isSome(byServiceCategory) ? `FILTER ( ?topic = svccat:${byServiceCategoryFilter})` : ''}
```

Comment:
It's indeed correct, but I would have seized the opportunity to chain options, with a `O.map` + a `O.getOrElse` that I fill is more functional. However, this decision ultimately comes down to personal preference. :wink: